routes/server.go

```diff
@@ -1287,6 +1287,13 @@ func (fes *APIServer) NewRouter() *muxtrace.Router {
 			fes.UnregisterAsValidator,
 			PublicAccess,
 		},
+		{
+			"GetValidatorByPublicKeyBase58Check",
+			[]string{"GET", "OPTIONS"},
+			RoutePathValidators + "/{publicKeyBase58Check:[0-9a-zA-Z]{54,55}}",
```

Comment:
We could probably use a better base58 check regex. Here are the regexes we're using in deso-js to verify that the pub key is of a valid form:
```
const regexMainnet = /^BC[1-9A-HJ-NP-Za-km-z]{53}$/;
const regexTestnet = /^tBC[1-9A-HJ-NP-Za-km-z]{51}$/;
```